src/app/example_configs/config.yaml

```diff
@@ -199,3 +201,12 @@ ui:
   icon_default: fa-beer
   name: Beer Garden
   auto_refresh: false
+ldap:
```

Comment:
Unless we have another use for LDAP, we should move this under `auth.authentication_handlers.ldap`

Reply:
complete